lib/Syntax/README.md

```diff
@@ -175,7 +175,7 @@ StructDeclSyntaxBuilder Builder;
 // We previously parsed a struct keyword, let's tell the builder to use it.
 Builder.useStructKeyword(StructKeyword);
 
-// Hm, we didn't see an identifier, but saw a left brace. Let's keep going.
```

Comment:
This is fine as written.

Reply:
The original uses a non-standard variant spelling.

Reply:
I can take this or leave it. 'Hm' is just as correct as 'Hmm'.

Reply:
Okay I’m outvoted, we’ll keep it as “Hm”.